src/components/button/Button.tsx

```diff
@@ -71,7 +71,9 @@ const getButtonVariables = ({
                       inset 0 0 0 2px ${borderColor};
               }
               &:focus:not(:focus-visible) {
-                  box-shadow: none;
+                  box-shadow: ${variant === 'secondary'
+                      ? `inset 0 0 0 2px rgba(255, 255, 255, 0.32), inset 0 0 0 2px ${borderColor}`
```

Comment:
Slight tweak, needs to be the default state border:

```suggestion
                      ? `inset 0 0 0 1px ${borderColor};`
```